FinSearchUnified/Tests/Bundle/SearchBundleFindologic/NavigationQueryBuilderTest.php

```diff
@@ -25,6 +25,7 @@ class NavigationQueryBuilderTest extends TestCase
 
     protected function setUp()
     {
+        $this->markTestSkipped(sprintf('Obsolete test class: %s', get_class()));
```

Comment:
If this is obsolete, you could also just remove it, instead of just not running the tests.